src/deluge/io/debug/print.cpp

```diff
@@ -27,11 +27,109 @@ extern "C" {
 
 namespace Debug {
 
+constexpr uint32_t kNumSamplesBetweenReports = 44100;
+
+constexpr bool SysExLoggingEnabled = true;
```

Comment:
can this be a pre processor define? It'll make it easier to configure

Reply:
I did add a #define SYSEX_LOGGING_ENABLED which is defaulted to false if not defined by the build configuration

Reply:
Thanks! I just want to make it easy to find if it needs to be turned on/off